docs/src/identifiability/identifiability.md

```diff
@@ -0,0 +1,12 @@
+# Assess All Types of Identifiability
+
+```@docs
+assess_identifiability
+find_identifiable_functions
```

Comment:
I suggest not to put it here for the time being, `find_identifiable_functions` functionality is not really well-developed and well-tested at the moment. There is work in progress, on this.